include/api.php

```diff
@@ -2782,7 +2783,7 @@ function api_direct_messages_new($type) {
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
 
 		if (!x($_POST, "text") OR (!x($_POST,"screen_name") AND !x($_POST,"user_id"))) return;
```

Comment:
Standards: Please add braces to this conditional statement, spaces after commas and change OR/AND to ||/&& since it seems to be the prevalent usage in the file.